src/client/components/Navbar.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import logo from "./../assets/images/logo.png";
+import AutoComplete from "./AutoComplete";
+import { autoCompleteData } from "./Data";
+
+export default function Navbar() {
+  return (
+    <div>
+      <nav className="navbar navbar-dark navbar-expand bg-primary fixed-top">
+        <div className="container">
+          <button
+            className="navbar-toggler"
+            type="button"
+            data-toggle="collapse"
+            data-target="#Navbar"
+          >
+            <span className="navbar-toggler-icon"></span>
+          </button>
+          <a className="navbar-brand" href="#">
```

Comment:
Appreciate your code logic. You can try to use <Link> tag instead of an  <a> tag.